index.js

```diff
@@ -89,7 +93,8 @@ module.exports = function obj2Osm (opts) {
     })
 
     var attr = filterProps(row, WHITELISTS[row.type].attributes)
-    next(null, h(row.type, attr, children))
+    var nodeName = row.type === 'observation' ? 'node' : row.type
```

Comment:
Could you add a comment for Future Us that explicitly says what this is for / about?